web/src/containers/Wallets/History.tsx

```diff
@@ -79,6 +82,10 @@ export class WalletTable extends React.Component<Props> {
         if (!currencies.length && nextProps.currencies.length) {
             this.props.fetchCurrencies();
         }
+
+        if (!withdrawSuccess && nextProps.withdrawSuccess) {
+            this.props.fetchHistory({ page: 0, currency, type, limit: 6 });
```

Comment:
can you please create a constant 'rowsPerPage' in this file and assign the value of this variable to 6?
Please use this value everywhere in this file